### PR TITLE
fixed misaligned 3d viewer components

### PR DESCRIPTION
**Pull request by Abse2001**
/claim #48 

image

### CODE REVIEW

File: src/ContainerWithTooltip.tsx [hunk 1]
```diff
@@ -19,28 +19,33 @@ const ContainerWithTooltip = ({
...
+        mouseRef.current.y,
+        0.5,
+      )
       vector.unproject(camera)
       setMousePosition([vector.x, vector.y, vector.z])
     }
   })
 
   const groupProps: GroupProps = {
-    position,
```
Comment (seveibar) [inline src/ContainerWithTooltip.tsx:37]: This is the  actual fix and the rest is formatting?

**Review by seveibar — APPROVED** [review]

File: src/ContainerWithTooltip.tsx (hunk 1 above)
Comment (Abse2001) [reply src/ContainerWithTooltip.tsx:37]: ya but the mousePosition is calcultaed wrong Im going to open an issue for it.

image

**Comment by Abse2001** [comment]
@Anshgrover23 is the pad under the resistor?

**Comment by Abse2001** [comment]
@Anshgrover23 oh yeah I didn't update the snippet so you will not see any change yet,
There is another problem in 3d viewer needs a fix so I may try to fix that before I push to pro

**Pull request merged**